jobs/tests.py

```diff
@@ -1,3 +1,28 @@
 from django.test import TestCase
 
-# Create your tests here.
+from model_mommy import mommy
+
+from jobs.models import Job
+
+
+class JobMethodTests(TestCase):
+
+	def setUp(self):
+		self.job = mommy.make(Job, ready_to_publish=True)
+
+	def test_is_published(self):
+		published_date_is_not_none = False
+		self.job.publish()
+		if self.job.published_date:
+			published_date_is_not_none = True
+		self.assertEqual(published_date_is_not_none, True)
+		print "Published date:{0}".format(self.job.published_date)
+
+	def test_expiration_date(self):
+		expiration_date_is_not_none = False
+		self.job.publish()
+		self.job.set_expiration_date()
+		if self.job.expiration_date:
+			expiration_date_is_not_none = True
+		self.assertEqual(expiration_date_is_not_none, True)
+		print "Expiration date:{0}".format(self.job.expiration_date)
```

Comment:
Also check other edge cases that you can think of - what happens if you try to set an expiration date when there's one already set, what if you try to set an expiration date on a job that's not ready to publish?